src/App.jsx

```diff
@@ -5,6 +5,7 @@ import SignUp from './components/SignUp';
 import Home from './components/Home';
 import ProtectedRoute from './components/Details/ProtectedRoute';
 import Index from './components/Index';
+import Reservations from './components/Reservations.jsx/Reservations';
```

Comment:
- [ ] Is better to avoid adding 'extensions' to a folder name, this will end up confusing about what are you accessing if the directory or the file. Please address this issue

Reply:
Hey @williamrolando88, I'm not quite sure what you mean here, could you clarify?

Reply:
In the directories, you named a folder with the same name as the file which contains it. If you end up importing manually a resource with this issue you can get confused about if you are accessing the folder or the file.
![image](https://user-images.githubusercontent.com/52635378/167467736-b0866b71-30f4-46de-b75c-b873c955bd7e.png)

Reply:
@williamrolando88, omg I didn't even realize I had put an extension on the folder name which is why I totally didn't get your point. 🙈 Ouch!